data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -179,6 +180,7 @@
     (CleanHeightAndWeight,),  # dependent on MeasurementRecordsSuppression
     (UnitNormalization,),  # dependent on CleanHeightAndWeight
     (DropZeroConceptIDs,),
+    (CreatePersonExtTable,),
```

Comment:
This should actually be in `DEID_BASE_CLEANING_CLASSES` so it can exist for both output datasets.

Reply:
moving